tools/websocket_to_posix_proxy/src/websocket_to_posix_proxy.c

```diff
@@ -100,20 +100,22 @@ void WebSocketMessageUnmaskPayload(uint8_t* payload,
   }
 }
 
-extern MUTEX_T webSocketSendLock;
```

Comment:
Why does declaring this as extern no longer work?

Reply:
That is because this file was changed to be a .c file, and the file that uses it is a .cpp file. It seems that the name mangling is different.

Oddly I tried do use `extern "C"` for the definition in the .cpp file, but that did not quite work. So I turned it into a function dispatch.